identity-service/src/routes/relay.js

```diff
@@ -49,7 +50,11 @@ module.exports = function (app) {
         }
       }
 
+      const reqIP = req.get('X-Forwarded-For')
```

Comment:
this pattern works. i think fine.

we do have some ~complex logic to get the real clientIP if we wanted to just pass that: https://github.com/AudiusProject/audius-protocol/blob/ef9e20365c61c8f96febfe70bccf4f36e4950835/identity-service/src/rateLimiter.js#L30

mostly as FYI, i think this is good. Maybe we should just pass the req through and pull the IP/header inside the detectAbuse fn?